EDEngineer/Resources/Data/entryData.json

```diff
@@ -1874,4 +1874,2476 @@
     ],
     "Group": "GuardianRuinsActive"
   }
-]
\ No newline at end of file
+]
+[

```

Comment:
so the new ingredients needs to be in the same array, you should just append the ingredients directly

Reply:
Will do.